packages/appium/docs/en/guides/migrating-1-to-2.md

```diff
@@ -184,7 +184,7 @@ For many years, some of Appium's authors maintained the [WD](https://github.com/
 
 ### :warning: Appium Inspector split out from Appium Desktop
 
-The inspecting portion of Appium Desktop has been moved to its own app, Appium Inspector: [github.com/appium/appium-inspector](https://github.com/appium/appium-inspector). It's fully compatible with Appium 2.0 servers. Simply download it and run it on its own. You no longer need the GUI Appium Desktop server to inspect apps. The Appium Desktop server will continue to be supported at its original site: [github.com/appium/appium-desktop](https://github.com/appium/appium-desktop). It will simply no longer bundle the Inspector with it.
+The inspecting portion of Appium Desktop has been moved to its own app, Appium Inspector: [github.com/appium/appium-inspector](https://github.com/appium/appium-inspector). It's fully compatible with Appium 2.0 servers. Simply download it and run it on its own. You no longer need the GUI Appium Desktop server to inspect apps. The Appium Desktop server will continue to be supported at its original site: [github.com/appium/appium-desktop](https://github.com/appium/appium-desktop). It will simply no longer bundle the Inspector with it. Note that the old Appium-Desktop's inspector feature depends on the deprecated [WD](https://github.com/admc/wd) client. It means the old one will not work for Appium 2.0.
```

Comment:
`Appium-Desktop's inspector feature` not sure what the `feature` is in that context

Reply:
i meant AD 1.21 and lower ones. The inspector uses WD. I saw a few times users used the old one for Appium 2, and failed to create a new session as no-w3c support